frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -23,7 +23,46 @@ const CourseAttribute = ({ title, content }) => {
   );
 };
 
-export default function CourseDescription() {
+const PlannerDropdown = ({courseCode, structure, addToPlanner}) => {
+  const [categories, setCategories] = useState([])
+
+  useEffect(() => {
+    // Example groups: Major, Minor, General
+    const categoriesDropdown = []
+    for (const group in structure) {
```

Comment:
we can leave it like this for now - but we might need to expand this to have an option to "double count"